styles/widgets/generic/gridBase.generic.less

```diff
@@ -737,6 +737,16 @@
                 }
             }
 
+            & .dx-command-edit {
+                background-color: ~"@{@{widgetName}-row-focused-bg}";
+                border-top: 1px solid ~"@{@{widgetName}-row-focused-bg}";
+                border-bottom: 1px solid ~"@{@{widgetName}-row-focused-bg}";
+
+                & .dx-link {
+                    color: ~"@{@{widgetName}-row-focused-color}";
+                }
+            }
```

Comment:
This styles is already defined in another place. So, this new styles are duplicated.